action.yml

```diff
@@ -56,6 +56,23 @@ description: |
   //pkgs.dev.azure.com/USXpress-Inc/CloudOps/_packaging/packages/npm/:email=devops@usxpress.com
   ; end auth token
   ```
+
+  ## Locating packages
+
+  Azure artifacts of NPM packages are located here.
+
+  ```text
+  https://dev.azure.com/USXpress-Inc/CloudOps/_artifacts/feed/packages
+  ```
+
+  ## Locating Container Images
+
+  ECR containers can be located with this URI.
+
+  ```text
+  064859874041.dkr.ecr.us-east-2.amazonaws.com/<octopusgroup>/<app name>
```

Comment:
I don't believe that it uses octopusgroup or app_name. It uses what you provide in ecr_repository. Please update in other repos as well